packages/lesswrong/themes/stylePiping.ts

```diff
@@ -256,7 +256,17 @@ const baseBodyStyles = (theme: ThemeType): JssStyles => ({
   },
   // CKEditor wraps tables in a figure element
   '& figure.table': {
-    display: 'table'
+    width: 'fit-content !important',
+    height: 'fit-content !important',
+  },
+  // Many column tables should overflow instead of squishing
+  //  - NB: As of Jan 2023, this does not work on firefox, so ff users will have
+  //    squishy tables (which is the default behavior above)
+  '& figure.table:has(> table > tbody > tr > td + td + td + td)': {
+    overflowX: 'auto',
+    '& table': {
+      width: 700,
+    },
```

Comment:
Very happy with this one. My (and Jonathan's) first use of a [parent selector](https://developer.mozilla.org/en-US/docs/Web/CSS/:has) and it's great.